app/Http/Controllers/LiquidationController.php

```
@@ -0,0 +1,81 @@
+        $data = PaymentRequest::liquidationRequest()
+            ->withStakeholder()
+            ->payment()
+            ->withTransactionFlow()
+            ->with('created_by_user')
+            ->paginate(config('services.pagination.limit'));
+        return response()->json([
+            'data' => $data
+        ], 200);
+    }
+
+    public function show($id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)